### PR TITLE
Show dates on Bilara translations in Suttaplex

### PR DESCRIPTION
**Pull request by thesunshade**
Fixes this issue: https://github.com/suttacentral/suttacentral/issues/2362

I have tested this on my local build.

image

image

@buddhist-uni is responsible for this

### CODE REVIEW

**Comment by khemarato** [comment]
Oh! That actually worked‽ Nice! 😁

**Comment by thesunshade** [comment]
The problem with Ven. Brahmalli's translations not working.

Ready to go now.

**Comment by thesunshade** [comment]
BTW, the server has to be restarted to get this to work.

File: server/server/common/queries.py [hunk 1]
```diff
@@ -556,14 +556,20 @@
                 FILTER doc.uid == v.uid AND 'comment' IN doc.muids AND author_doc.uid in doc.muids
                 RETURN doc.muids
             )
+            LET publication = (
+                FOR publication IN publications_v2
+                FILTER publication.creator_uid == text.muids[2]
+                AND STARTS_WITH(text.uid, SUBSTITUTE(publication.text_uid, "pli-tv-vi", "pli-tv-"))
+                RETURN publication
+            )[0]
```
[the remark this answers is not in the document]
Comment (khemarato) [reply server/server/common/queries.py:564]: I believe in AQL it just returns null (though @sourcery-ai is right that an index out of bounds error would be thrown in many other languages)

File: server/server/common/queries.py [hunk 2]
```diff
@@ -556,14 +556,20 @@
...
             RETURN {
                 lang: text.lang,
                 lang_name: lang_doc.name,
                 is_root: lang_doc.is_root,
                 author: author_doc.long_name,
                 author_short: author_doc.short_name,
                 author_uid: author_doc.uid,
-                publication_date: null,
+                publication_date: publication.first_published,
```
[the remark this answers is not in the document]
Comment (khemarato) [reply server/server/common/queries.py:572]: This is an AQL string, not Python, Mr. Robot -_-

**Comment by ihongda** [comment]
👍🏼 Thanks @thesunshade @buddhist-uni

**Pull request merged**